frontend-implicaction/src/app/admin/users/components/users-table/table-users.component.ts

```diff
@@ -14,6 +15,8 @@ export class TableUsersComponent {
 
   loading = true; // indique si les données sont en chargement
 
+  selectedUsers: User[];
```

Comment:
```suggestion
  selectedUsers: User[] = [];
```
ça permet d'éviter un `cannot find xx of undefined`